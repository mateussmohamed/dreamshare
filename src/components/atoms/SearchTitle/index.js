import styled from 'styled-components'

const SearchTitle = styled.h2`
  ${props => props.theme.typography.montserratStyle};
  font-size: 22px;
  font-weight: bold;
  text-align: center;
  color: ${props => props.theme.colors.greyishBrown};
`

export default SearchTitle
