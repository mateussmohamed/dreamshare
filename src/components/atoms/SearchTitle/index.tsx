import styled from 'styled-components'

export const SearchTitle = styled.h2`
  ${props => props.theme.typography.montserratStyle};
  font-size: 22px;
  font-weight: bold;
  text-align: center;
  color: ${props => props.theme.colors.greyishBrown};
`
