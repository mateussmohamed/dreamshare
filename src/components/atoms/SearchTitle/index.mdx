---
name: SearchTitle
menu: Atoms
---

import { Playground, PropsTable } from 'docz'
import SearchTitle from './index'

# SearchTitle
<PropsTable of={SearchTitle} />

<Playground>
  <SearchTitle>
    SHARE YOUR HOLIDAY DREAM
  </SearchTitle>
</Playground>
