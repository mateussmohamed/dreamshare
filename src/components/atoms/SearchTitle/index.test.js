import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import SearchTitle from './index'

describe('[<SearchTitle />]', () => {
  const wrap = (props = {}) => shallow(<SearchTitle {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap({ children: 'I am a Title' })

    expect(wrapper).toMatchSnapshot()
  })
})
