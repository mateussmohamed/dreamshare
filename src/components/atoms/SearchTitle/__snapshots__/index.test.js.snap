// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<SearchTitle />] matches to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "SearchTitle-sc-1b05g7-0",
        "isStatic": false,
        "rules": Array [
          [Function],
          ";font-size:22px;font-weight:bold;text-align:center;color:",
          [Function],
          ";",
        ],
      },
      "displayName": "SearchTitle",
      "render": [Function],
      "styledComponentId": "SearchTitle-sc-1b05g7-0",
      "target": "h2",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  I am a Title
</StyledComponent>
`;
